frontend/locales/en.json

```diff
@@ -37,6 +37,13 @@
   "chooser": {
     "max_value_reached": "Value too large!"
   },
+  "generate-room-code": {
```

Comment:
We agreed on the naming convention of underscores for spaces so please format these entries that way!